audit-service/src/unit-tests/java/uk/gov/gchq/palisade/service/audit/service/SimpleAuditServiceTest.java

```diff
@@ -53,41 +49,38 @@ public void setUp() {
         logger.addAppender(appender);
     }
 
-    @After
+    @AfterEach
     public void tearDown() {
         logger.detachAppender(appender);
         appender.stop();
     }
 
-    //@DataPoints requires the property to be public
-    @SuppressWarnings("checkstyle:visibilitymodifier")
-    @DataPoints()
-    public static AuditRequest[] requests = new AuditRequest[] {
-            registerRequestCompleteAuditRequest(),
-            registerRequestExceptionAuditRequestFromUserService(),
-            readRequestCompleteAuditRequest(),
-            readRequestExceptionAuditRequest()
-
-    };
-
     private List<String> getMessages(final Predicate<ILoggingEvent> predicate) {
         return appender.list.stream()
                 .filter(predicate)
                 .map(ILoggingEvent::getFormattedMessage)
                 .collect(Collectors.toList());
     }
 
-    @Theory
-    public void auditOnAuditRequest(final AuditRequest request) {
+    @Test
+    void testAuditErrorMessage() {
         // When
-        auditService.audit(request);
+        auditService.audit(ApplicationTestData.TEST_TOKEN, ApplicationTestData.auditErrorMessage("USER_SERVICE"));
 
         // Then
         List<String> logMessages = getMessages(event -> true);
 
-        MatcherAssert.assertThat(logMessages, Matchers.hasItems(
-                Matchers.containsString(request.toString())
-        ));
+        assertThat(logMessages.get(0)).contains("SimpleAuditService.audit");
     }
 
+    @Test
+    void testAuditSuccessMessage() {
+        // When
+        auditService.audit(ApplicationTestData.TEST_TOKEN, ApplicationTestData.auditSuccessMessage("DATA_SERVICE"));
+
+        // Then
+        List<String> logMessages = getMessages(event -> true);
+
+        assertThat(logMessages.get(0)).contains("SimpleAuditService.audit");
```

Comment:
Your assertions are the same, and dont indicate success or failure.
A better assertion for each would be
```suggestion
        assertThat(logMessages.get(0)).contains(AuditSuccessMessage.class.getSimpleName());
```

Reply:
Done